app/model/db/idx_token_list.py

```diff
@@ -29,7 +29,7 @@ class IDXTokenListItem(Base):
     # Token Address
     token_address = Column(String(42), primary_key=True)
     # Token Template
-    token_template = Column(String(40), nullable=False)
+    token_template = Column(String(40))
```

Comment:
I found that token_template NOT NULL restriction is not created by legacy migration files, so fixed model.